app/src/main/java/com/google/droidjump/GameFragment.java

```diff
@@ -48,19 +54,51 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         View rootView = inflater.inflate(R.layout.game_screen, container, /* attachToRoot= */ false);
         ((LinearLayout) rootView.findViewById(R.id.game_layout)).addView(gameView);
-        NavigationHelper.addOnBackPressedEventListener(activity, new StartFragment());
+        menuButton = rootView.findViewById(R.id.menu_button);
+        pauseLayout = rootView.findViewById(R.id.pause_layout);
+        menuButton.setOnClickListener(ignored -> {
```

Comment:
nit: you can probably remove the brackets here and below:
```
menuButton.setOnClickListener(ignored -> onPause());
```

Reply:
Fixed.